### ISSUE TITLE
Created a related plugin - Link somewhere?

### ISSUE THREAD

**Issue by ahalota** [calvinmetcalf/leaflet.shapefile#43]
I made a small plugin that uses your two plugins that is probably of interest to people using this one. I haven't done much testing on it, so I don't want to submit it to main leaflet docs.

This plugin is an extension to the Leaflet.draw toolbar that adds an extra button for importing a shapefile.
[Leaflet.draw-extension](https://github.com/ahalota/Leaflet.draw-extension)

[Demo](https://ahalota.github.io/Leaflet.draw-extension/demo.html)

Is there a good place on github to post things like this?

**Comment by calvinmetcalf** [comment]
yes you can add it to the [leaflet-plugin-page](http://leafletjs.com/plugins.html#develop-your-own)
